hack/update-template/main.go

```diff
@@ -0,0 +1,212 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"reflect"
+	"strings"
+
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	maxDepth           = 1
+	deployFolderPath   = "../../deploy/"
+	outputTemplateFile = "../../openshift/template.yaml"
+)
+
+var (
+	// this holds all of the info that is static in the template. update this as needed
+	saasTemplateFile = Template{
+		APIVersion: "template.openshift.io/v1",
+		Kind:       "Template",
+		Metadata: Metadata{
+			Name: "configuration-anomaly-detection-template",
+		},
+		Parameters: []Parameter{
+			{Name: "IMAGE_DIGEST", Value: "sha256:22d2a957d935e883f45addd35acc87450ce71ea2f94c33f2df70ff36b015486a"},
+			{Name: "REGISTRY_IMG", Value: "quay.io/app-sre/cadctl"},
+			{Name: "NAMESPACE_NAME", Value: "configuration-anomaly-detection"},
+		},
+	}
+)
+
+func main() {
+	ex, getExecutableErr := os.Executable()
+	if getExecutableErr != nil {
+		panic(fmt.Errorf("could not get the Executable: %w", getExecutableErr))
+	}
+
+	workingDirectory := filepath.Dir(ex)
+	fmt.Printf("workingDirectory :: %s\n", workingDirectory)
```

Comment:
Any reason not to do `os.Getcwd()` here?

Reply:
Yeah, it is where the caller is, not where the script is